ksqldb-udf/pom.xml

```diff
@@ -36,12 +36,6 @@
             <version>${project.version}</version>
             <scope>test</scope>
         </dependency>
-        <dependency>
-            <groupId>com.google.errorprone</groupId>
-            <artifactId>error_prone_annotations</artifactId>
-            <version>2.2.0</version>
-            <scope>compile</scope>
-        </dependency>
         <dependency>
```

Comment:
In this case, I had to remove the explicit dependency because it was too old. The nearest transitive one is the correct one.